Makefile

```diff
@@ -29,10 +29,6 @@ build:  ## Build binary
 	go generate ./...
 	go build -v -ldflags="-w -s -X github.com/0xERR0R/blocky/util.Version=${VERSION} -X github.com/0xERR0R/blocky/util.BuildTime=${BUILD_TIME}" -o $(BIN_OUT_DIR)/$(BINARY_NAME)$(BINARY_SUFFIX)
 
-build-static:  ## Build static binary
-	go generate ./...
-	go build -tags static -v -ldflags="-linkmode external -extldflags -static -X github.com/0xERR0R/blocky/util.Version=${VERSION} -X github.com/0xERR0R/blocky/util.BuildTime=${BUILD_TIME}" -o $(BIN_OUT_DIR)/$(BINARY_NAME)$(BINARY_SUFFIX)
```

Comment:
As said in my other comments, I think it'd be nicer to consolidate everything related to building blocky itself in the `Makefile`.  
Here's a minimal one that should be enough to cover all uses and avoid repeating the `-X` flags in different places:
```Makefile
VERSION := $(shell git describe --always --tags)
BUILD_TIME := $(shell date '+%Y%m%d-%H%M%S')
ARCHITECTURE := $(shell go env GOARCH)

DOCKER_IMAGE_NAME := spx01/blocky
BINARY_NAME := blocky
BIN_OUT_DIR := bin

GO_BUILD_FLAGS := -v

GO_BUILD_LD_FLAGS := \
	-w \
	-s \
	-X github.com/0xERR0R/blocky/util.Version=${VERSION} \
	-X github.com/0xERR0R/blocky/util.BuildTime=${BUILD_TIME} \
	-X github.com/0xERR0R/blocky/util.Architecture=${ARCHITECTURE}

GO_BUILD_OUTPUT := $(BIN_OUT_DIR)/$(BINARY_NAME)$(BINARY_SUFFIX)

build: ## Build binary
	go generate ./...
	go build \
		${GO_BUILD_FLAGS} \
		-ldflags="${GO_BUILD_LD_FLAGS}" \
		-o ${GO_BUILD_OUTPUT}

build-static: GO_BUILD_FLAGS += -tags static -linkmode external -extldflags -static
build-static: build ## Build static binary
```

From the `Dockerfile`: `make build-static GO_BUILD_OUTPUT=/bin/blocky`

Also a couple of tiny pre-existing issues with the `Makefile` we can probably fix at the same time:

 - `build-static` is missing from `.PHONY`
 - the shebang is wrong and should be removed

Reply:
build-static was never intended for user usage. I only added it to run inside the Docker image creation.
The suggested changes would most likely create broken binaries during cross compilation.
To get propper images I relay on zig inside the Dockerfile.